### ISSUE TITLE
Replication clarifications

### ISSUE THREAD

**Issue by silviucpp** [Netflix/dynomite#669]
Hello,

I'm using the following config on the nodes (of course tokens are different and some ip's):

```bash
dyn_o_mite:
  datacenter: dc01
  rack: rack01
  dyn_listen: 172.17.33.179:8101
  dyn_seed_provider: simple_provider
  dyn_seeds:
  - dynomite01dc01rack01.bucharest.nbz:8101:rack01:dc01:1431655765
  - dynomite01dc01rack02.bucharest.nbz:8101:rack02:dc01:1431655765
  - dynomite02dc01rack01.bucharest.nbz:8101:rack01:dc01:2863311530
  - dynomite02dc01rack02.bucharest.nbz:8101:rack02:dc01:2863311530
  - dynomite03dc01rack01.bucharest.nbz:8101:rack01:dc01:4294967295
  - dynomite03dc01rack02.bucharest.nbz:8101:rack02:dc01:4294967295

  listen: 172.17.33.179:8102
  servers:
  - 127.0.0.1:6379:1
  tokens: '1431655765'
  secure_server_option: datacenter
  pem_key_file: /etc/dynomite/dynomite.pem
  recon_key_file: /etc/dynomite/recon_key.pem
  recon_iv_file: /etc/dynomite/recon_iv.pem
  data_store: 0
  stats_listen: 172.17.33.179:22222
  max_msgs: 200000
  mbuf_size: 16384
  write_consistency: DC_ONE
  read_consistency: DC_ONE
```

I have the following scenario: 1 DC and 2 racks, each one having 3 nodes. 

Let's suppose key X is distributed to node 2 on each rack. 

Doing the following test:

1. Stop the redis process on the node2 rack1 
2. Send the request: `SET X value1` on node2 rack2 will succedd
3. Start back the redis process on node2 rack1 
4. Run the `GET X` on rack1 node2 (no key found). run same command on the other rack key is found there.

Seems when node from the first rack comes online the writes did while was down are not replicated ? Is this expected behaviour ? I was expecting when node comes back the writes during the downtime to be replicated by the peer on that node. (like in cassandra)

Another scenario:

1. Stop the dynomite process on the node2 rack1 
2. Send the request: `SET X value1` on node2 rack2 will return <<"ERR Peer: Peer Node is not connected">>

So seems if a peer node is down you can't write ... I was expecting that write works fine and when peer comes online all those to be replicated there as well.

Anything wrong in my config or the replication works only all nodes are online at the moment you write the key ?

Silviu

**Comment by silviucpp** [comment]
Ohh so my problem is similar with https://github.com/Netflix/dynomite/issues/655 . Seems using DM is mandatory for this use-case.

**Comment by silviucpp** [comment]
I tried another scenario:



I blocked the connection in firewall between 2 peer nodes that stores the same token range . Node A in rack 1 and node B from rack 2.

Then I write a key on rack1 node . and read from both nodes. node from rack 1 has the last value the other one no. Expected.

If I remove the firewall restrictions in less than 25 seconds or something like this. The value is replicated also on rack2 node. But if this connection restriction in firewall stays longer the nodes never come in sync.
